specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -62,7 +152,7 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/CheckNameAvailabilityResultResource"
+              "$ref": "#/definitions/CheckNameAvailabilityResult"
```

Comment:
>CheckNameAvailabilityResult [](start = 37, length = 27)

I believe this is a breaking change but I'll let the SDK reviewers comment

Reply:
Yes @pilor ,  the earlier schema for response was erronous and we are fixing it with this pull request. It is in accordance with format mentioned at - https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#check-name-availability-requests

We didn't publish an SDK with the erroneous swagger, hence it shouldn't cause an issue after checking in this PR